plugin-template

```diff
@@ -66,6 +66,7 @@ DEFAULT_SETTINGS = {
     "test_cli": False,
     "test_performance": False,
     "test_released_plugin_with_next_pulpcore_release": False,
+    "disabled_redis_runners": [],
```

Comment:
Wouldn't it be the easiest to just default to "s3" here? It shouldn't fail if "s3" wasn't even tested, no?

Reply:
It should not fail. Yet, I think that we should not disable Redis for any storage by default.